### PR TITLE
Sbt auto plugin

### PR DESCRIPTION
**Pull request by yasuba**
Upgrading scalapact-sbtplugin build.sbt to allow multi-module projects to work

### CODE REVIEW

File: scalapact-sbtplugin/src/main/scala/com/itv/scalapact/plugin/ScalaPactPlugin.scala [hunk 1]
```diff
@@ -9,30 +9,32 @@ import sbt._
...
+  import autoImport._
 
   private val pactSettings = Seq(
     providerStateMatcher := PartialFunction { (_: String) => false },
     providerStates := Seq(),
-    pactBrokerAddress := "",
-    providerName := "",
     consumerNames := Seq.empty[String],
     versionedConsumerNames := Seq.empty[(String,String)],
-    pactContractVersion := "",
```
Comment (davesmith00000) [inline scalapact-sbtplugin/src/main/scala/com/itv/scalapact/plugin/ScalaPactPlugin.scala]: Setting the defaults to blank always bothered me so this looks like a good idea, but have you checked if the rest of the code does anything like `if(pactBrokerAddress.isEmpty) ...`?

Comment (davesmith00000) [inline scalapact-sbtplugin/src/main/scala/com/itv/scalapact/plugin/ScalaPactPlugin.scala]: If the default is still being set that's fine but I've missed where.

**Comment by yasuba** [comment]
I'm able to publish pacts with the default settings replaced - so I've reinstated them.

**Comment by davesmith00000** [comment]
Thanks! Approved.

**Pull request merged**